adapters/dmx/dmx.go

```diff
@@ -31,12 +31,21 @@ type dmxExt struct {
 	Bidder dmxParams `json:"bidder"`
 }
 
+type dmxPubExt struct {
+	Dmx dmxPubExtId `json:"dmx,omitempty"`
+}
+
+type dmxPubExtId struct {
+	Id string `json:"id,omitempty"`
+}
+
 type dmxParams struct {
-	TagId       string `json:"tagid,omitempty"`
-	DmxId       string `json:"dmxid,omitempty"`
-	MemberId    string `json:"memberid,omitempty"`
-	PublisherId string `json:"publisher_id,omitempty"`
-	SellerId    string `json:"seller_id,omitempty"`
+	TagId       string  `json:"tagid,omitempty"`
+	DmxId       string  `json:"dmxid,omitempty"`
+	MemberId    string  `json:"memberid,omitempty"`
+	PublisherId string  `json:"publisher_id,omitempty"`
+	SellerId    string  `json:"seller_id,omitempty"`
+	Bidfloor    float64 `json:"bidfloor,omitempty"`
```

Comment:
We do recommend moving to using the bid floor from the openrtb request rather than having a custom parameter in your bidder parameters. Publishers prefer having one consistent location to control their bid floors rather than spamming them across all the bidder parameters.